operate/services/manage.py

```diff
@@ -2061,11 +2091,117 @@ def refill_requirements(  # pylint: disable=too-many-locals
 
         return {
             "balances": balances,
+            "bonded_olas": bonded_olas,
             "refill_requirements": refill_requirements,
             "is_refill_required": is_refill_required,
             "allow_start_agent": allow_start_agent,
         }
 
+    def _compute_bonded_olas(  # pylint: disable=too-many-locals
```

Comment:
@kupermind Please review this method. The output should be the currently bonded olas as agent bonds + security deposit.